lte/gateway/c/session_manager/LocalEnforcer.h

```diff
@@ -713,6 +713,29 @@ class LocalEnforcer {
    * @param dead_sessions_to_cleanup
    */
   void cleanup_dead_sessions(const RuleRecordSet dead_sessions_to_cleanup);
+
+  /**
+   * Given a UE id and session, add the session to the session map,
+   * and invoke API that adds shard to shard tracker if the UE doesn't already
+   * exist in the session map. If the UE already exists, set the shard id of
+   * the session to the same as that of another session of the UE.
+   * @param imsi id of UE
+   * @param session_map map of UE to vector of session states
```

Comment:
Is the SessionState passed into this function included in the session map? Or does it not matter for this function?

Reply:
It is included in session map, this is just adding it to the shard. It has to be included in order to set the shard id.